examples/react-native/__tests__/js/app-services/app-provider.test.jsx

```diff
@@ -0,0 +1,35 @@
+// :snippet-start: app-provider
+import React from 'react';
+import {AppProvider} from '@realm/react';
+// :remove-start:
+import 'react-native';
+import {render} from '@testing-library/react-native';
+import {useApp} from '@realm/react';
+import {View, Text} from 'react-native';
+const APP_ID = 'example-testers-kvjdy';
+function MyApp() {
```

Comment:
```suggestion
import {View, Text} from 'react-native';

const APP_ID = 'example-testers-kvjdy';

function MyApp() {
```
It's a bit more readable when there's a new line between blocks of imports, consts, functions, etc.

Reply:
Applies elsewhere